src/applications/registry.json

```diff
@@ -1002,6 +1002,15 @@
       "layout": "page-react.html"
     }
   },
+  {
+    "appName": "covid-vaccine-expansion",
```

Comment:
The `appName` is used as the title of the HTML page that is generated. Do we want to update the formatting to read `Covid Vaccine Expansion`?